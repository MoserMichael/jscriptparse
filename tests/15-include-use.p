    use "tests/testuse.p"
    
    c = newComplex(2,3)
    d = newComplex(4,5)
    e = cmul(c, d)
    println("complex product: {cshow(e)}")
