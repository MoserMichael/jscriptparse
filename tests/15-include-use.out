complex product: -7+i22
